main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -743,8 +739,8 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
     def Primary: Rule1[ParsedAst.Expression] = rule {
       Static | Scope | LetMatch | LetMatchStar | LetRecDef | LetUse | LetImport | IfThenElse | Reify | ReifyBool |
         ReifyType | ReifyPurity | Choose | TypeMatch | Match | LambdaMatch | Try | Lambda | Tuple |
-        RecordOperation | RecordLiteral | Block | RecordSelectLambda | NewChannel |
```

Comment:
Is `PutChannel` already gone?

Reply:
PutChannel was elsewhere in the grammar (not primary).